spec/features/user_peep_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'user peep' do
+
+    background {visit '/'}
+
+    scenario 'Send Peep path works' do
+      visit '/send_peep'
+      expect(page.status_code).to eq 200
+    end
+  end
```

Comment:
Lint/BlockAlignment: `end` at 9, 2 is not aligned with `feature 'user peep' do` at 1, 0